Community-Supported/tde-to-hyper/tde_to_hyper.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+from pathlib import Path
+from hyperapi import HyperProcess, Telemetry, \
+    Connection, CreateMode, TableName, TableDefinition, \
+    escape_name, escape_string_literal, HyperException
+
+def convert_tde_to_hyper(tde_path: Path):
+    # Rename path with a .hyper extension in the directory of the tde file
+    hyper_database = tde_path.with_name(tde_path.stem + '.hyper')
+
+    with HyperProcess(telemetry=Telemetry.SEND_USAGE_DATA_TO_TABLEAU) as hyper:
+        with Connection(endpoint=hyper.endpoint, database=hyper_database, create_mode=CreateMode.CREATE_AND_REPLACE) as connection:
+            # Schema and table for TDE file is constant
+            schema = 'Extract'
+            table = 'Extract'
+            # Create the temp external table for the TDE file
+            create_external_table_query = _get_external_table_query(str(tde_path), schema, table)
+            connection.execute_command(create_external_table_query)
+
+            # Get the name of the table created from the catalog
+            td = connection.catalog.get_table_definition(table)
+            schema_table = f"{schema}.{table}" 
+
+            # Create the destination table in the Hyper database
+            create_table_command = f"CREATE TABLE {escape_name(schema_table)} AS SELECT * FROM {td.table_name}"

```

Comment:
this is incorrect. It creates the table `"public"."Extract.Extract"`, i.e. you are creating a table named `Extract.Extract` inside the namespace `public`.

You want to create a table named `Extract` inside the namespace `Extract`, though...